clients/drcachesim/tools/schedule_stats.cpp

```diff
@@ -140,7 +140,7 @@ schedule_stats_t::parallel_shard_memref(void *shard_data, const memref_t &memref
     static constexpr char THREAD_SEPARATOR = ',';
     static constexpr char WAIT_SYMBOL = '-';
     per_shard_t *shard = reinterpret_cast<per_shard_t *>(shard_data);
-    if (knob_verbose_ >= 4) {
+    if (knob_verbose_ >= 4 && shard->stream->get_input_interface() != nullptr) {
```

Comment:
Add comment that this condition is for tests, or better yet, support this in the test (see other comment).

Reply:
Removed.